Enable horizontal-multivec colorbar (#836)
higlass/higlass: CHANGELOG.md

## Next

- Prettified JS code

> - Added colorbar options `colorbarPosition` and `colorbarBackgroundColor` for `horizontal-multivec` track, enabling the colorbar slider for that track type.

Reviewer: Stupid nitpick

```suggestion
- Enabled the colorbar by slider by adding the options `colorbarPosition` and `colorbarBackgroundColor` for the `horizontal-multivec`
```
